CONTRIBUTING.md

```diff
@@ -0,0 +1,115 @@
+# [<img src="https://avatars1.githubusercontent.com/u/3771091?v=3&s=30">](https://github.com/gonum) Contribution Guidelines
+#### Introduction
+
+The Gonum project provides general purpose numerical computing and data science libraries for the Go programming language, and we would like you to join us in improving Gonum's quality and scope.
+This document is for anyone who is contributing or interested in contributing.
+
+#### Table of Contents
+
+[Getting Started](#getting-started)
+  * [Working Together](#working-together)
+  * [Project Goals](#project-goals)
+  * [The Gonum Repos](#the-gonum-repos)
+
+[How to Contribute](#how-to-contribute)
+  * [Reporting Bugs](#reporting-bugs)
+  * [Suggesting Enhancements](#suggesting-enhancements)
+  * [Your First Code Contribution](#your-first-code-contribution)
+  * [Pull Requests](#pull-requests)
+  * [Style](#style)
+
+## Getting Started
+
+Gonum discussion takes place on the [gonum-dev google group](https://groups.google.com/forum/#!forum/gonum-dev).
+
+### Working Together
+
+When contributing or otherwise participating, please:
+
+- Be friendly and welcoming
+- Be patient
+- Be thoughtful
+- Be respectful
+- Be charitable
+- Avoid destructive behavior
+
+Excerpted from the [Go conduct document](https://golang.org/conduct).
+
+### Project Scope
+
+The purpose of the Gonum project is to provide general purpose numerical computing and data science libraries for the Go programming language.
+The libraries should aim to provide building blocks for disciplinary work and advanced algorithms.
+Code should be implemented in pure Go.
+Calls to assembly, C, Fortran, or other languages may be justified with performance considerations, but should be opt-in for users.
+
+### The Gonum Repos
+
+Here are the current repos for the gonum project.
+If code you want to contribute doesn't quite fit in any of them, then please start a discussion on the [mailing list](https://groups.google.com/forum/#!forum/gonum-dev).
+
+* [matrix](https://github.com/gonum/matrix) - Matrix packages for the Go language.
```

Comment:
I think the package description don't have to follow the descriptions on GitHub. Especially "Go" and "the Go language" can be omitted here.